src/python/pants/backend/jvm/subsystems/rsc_native_image.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.jvm.subsystems.zinc import _ZINC_COMPILER_VERSION
```

Comment:
This probably isn't necessary since the versions are pretty independent, but it wasn't clear what a good default value was for this.

Reply:
Yea, these versions have nothing to do with one another: this tool should have a different default that corresponds to the version of rsc that we are using.